changelog.d/1142.added.md

```diff
@@ -0,0 +1 @@
+Include  into  (zipapp) so that it can be executed with Python 3.10 or earlier
```

Comment:
```suggestion
Include `tomli` into `pipx.pyz` (zipapp) so that it can be executed with Python 3.10 or earlier
```

Looks like there was some trouble with interpreting backticks?

Reply:
Yeah that's something to be careful of for sure. Will drop all the fragments once the changelog bump is merged and I can rebase.